src/patternfly/components/Table/table-tr.hbs

```diff
@@ -26,14 +27,20 @@
       hidden
     {{/if}}
   {{/if}}
-  {{#if table--grid}}
-    role="row"
-  {{/if}}
+  {{#unless table-tr--IsBorderRow}}
+    {{#if table--grid}}
+      role="row"
+    {{/if}}
+  {{/unless}}
   {{#if table-tr--IsHoverable}}
     tabindex="0"
   {{/if}}
   {{#if table-tr--attribute}}
     {{{table-tr--attribute}}}
   {{/if}}>
-  {{> @partial-block}}
+  {{#if table-tr--IsBorderRow}}
+    {{> table-td table-td--IsEmpty="true" table-td--attribute='colspan="100%" aria-hidden="true" tabindex="-1"'}}
```

Comment:
> `colspan="100%"`

Is that valid? From MDN, looks like this needs to be an integer https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attributes